python/examples/advanced/String_Tracking.ipynb

```diff
@@ -0,0 +1,767 @@
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>onlyDigits</th>\n",
+       "      <th>onlyAlpha</th>\n",
+       "      <th>mixed</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>12</td>\n",
+       "      <td>Alice</td>\n",
+       "      <td>my_email_1989@gmail.com</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>83</td>\n",
+       "      <td>Bob</td>\n",
+       "      <td>ADK-1171</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>1</td>\n",
+       "      <td>Chelsea</td>\n",
+       "      <td>Copacabana 272 - Rio de Janeiro</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>992</td>\n",
+       "      <td>Danny</td>\n",
+       "      <td>21º C Friday - Sao Paulo, Brasil</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>7</td>\n",
+       "      <td>Eddie</td>\n",
+       "      <td>18127819ASW</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  onlyDigits onlyAlpha                             mixed\n",
+       "0         12     Alice           my_email_1989@gmail.com\n",
+       "1         83       Bob                          ADK-1171\n",
+       "2          1   Chelsea   Copacabana 272 - Rio de Janeiro\n",
+       "3        992     Danny  21º C Friday - Sao Paulo, Brasil\n",
+       "4          7     Eddie                       18127819ASW"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import whylogs as why\n",
+    "import pandas as pd\n",
+    "data = {\n",
+    "    \"onlyDigits\": [\"12\", \"83\", \"1\", \"992\", \"7\"],\n",
+    "    \"onlyAlpha\": [\"Alice\", \"Bob\", \"Chelsea\", \"Danny\", \"Eddie\"],\n",
+    "    \"mixed\": [\"my_email_1989@gmail.com\",\"ADK-1171\",\"Copacabana 272 - Rio de Janeiro\",\"21º C Friday - Sao Paulo, Brasil\",\"18127819ASW\"]\n",
+    "}\n",
+    "df = pd.DataFrame(data)\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configuring the Metrics in the DatasetSchema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "whylogs uses `Resolvers` in order to define how a column name or data type gets mapped to different metrics.\n",
```

Comment:
`s/how a column name or data type gets mapped to different metrics/the set of metrics tracked for a column name or data type/`

Reply:
done!